README.md

```diff
@@ -23,7 +25,9 @@ Want to deploy immediately? Click this button
 
 [![Deploy to Netlify Button](https://www.netlify.com/img/deploy/button.svg)](https://app.netlify.com/start/deploy?repository=https://github.com/netlify-templates/nuxt-toolbox)
 
-To deploy using the Netlify cli:
+(Clicking this button will create a new repo for you that looks exactly like this one, and sets that repo up immediately for deployment on Netlify)
+
+### To deploy using the [Netlify cli](https://cli.netlify.com/):
 
```

Comment:
```suggestion

Either clone this repo or use the 'Template' button at the top of this repo. Then install the Netlify CLI tool and run `netlify init` or use the `netlify sites:create-template` command in you terminal ([learn more about this command here](https://www.netlify.com/blog/create-a-site-from-a-template-using-the-netlify-cli)).

```

Reply:
My suggestion to the suggestion (😆) is only saying `Click 'Use this Template' button at the top of this repo` because if folks do clone this repo, they are going to make PRs against our project. Otherwise this is great. I think giving them multiple options is great!

```suggestion
Click the 'Use the Template' button at the top of this repo. Then install the Netlify CLI tool and run `netlify init`. Or straight from the Netlify CLI, use the `netlify sites:create-template` command in you terminal ([learn more about this command here](https://www.netlify.com/blog/create-a-site-from-a-template-using-the-netlify-cli)) to do the entire flow for you.
```